src/nav/nav_model/nav_model_rings.py

```
@@ -33,3 +48,875 @@ def __init__(self,
+            self._logger.warning(
+                f'Could not compute edge radii for {edge_type} edge of '
+                f'{feature_name}')
+            return
+
+        edge_radii = edge_radii_bp.mvals
+        edge_a = self._get_base_radius(edge_data)
+
+        if edge_a is None:
+            self._logger.warning(
+                f'Could not get base radius for {edge_type} edge of '
+                f'{feature_name}')
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)